frontend/public/components/monitoring/alerting.tsx

```diff
@@ -548,7 +557,7 @@ const SilenceTableRow: React.FC<SilenceTableRowProps> = ({ obj, showCheckbox })
     [id, setSelectedSilences],
   );
 
-  const [namespace] = useActiveNamespace();
+  const { ns: namespace } = useParams();
```

Comment:
@tvu20 Could you please explain why we can't / prefer not to use `useActiveNamespace`?

Reply:
@kyoto should this be causing any issues ?

Reply:
No issues that I'm aware of. I was just wondering what motivated the change.